src/assets/javascript/analytics/ua/init.js

```diff
@@ -5,7 +5,7 @@ window.DI.analyticsUa = window.DI.analyticsUa || {};
   "use strict";
 
   function initGtm() {
-    const sendData = window.DI.core.sendData;
+    const sendData = window.DI.analyticsGa4.pageViewTracker.pushToDataLayer;
```

Comment:
If this is using the new https://www.npmjs.com/package/@govuk-one-login/one-login-analytics package, this should be added as a [peer dependency](https://docs.npmjs.com/cli/v10/configuring-npm/package-json#peerdependencies) and the PR and `README.md` updated to describe the breaking change required to use this new package.